Dockerfile

```diff
@@ -221,6 +221,7 @@ RUN useradd --comment 'Islandora User' --no-create-home -d /var/www/html --syste
     chgrp www-data /var/log/fits && \
     chmod 775 /var/log/fits && \
     sed -i 's#log4j.appender.FILE.File = .*#log4j.appender.FILE.File = /var/log/fits/fits.log#' /usr/local/fits/log4j.properties && \
+    sed -ie 's/<tool class="edu\.harvard\.hul\.ois\.fits\.tools\.tika\.TikaTool" exclude-exts="jar,avi,mov,mpg,mpeg,mkv,mp4,mpeg4,m2ts,mxf,ogv,mj2,divx,dv,m4v,m2v,ismv" classpath-dirs="lib\/tika"\/>/<!-- & -->/' /usr/local/fits/xml/fits.xml && \
```

Comment:
Hi @shauntru, 
This change looks great. Can you please add a _brief_ one line comment above this line detailing what it's fixing? Something to the effect of "commenting out this line keeps the tmp directory from filling up" with a link to the issue it's fixing. 
Thanks!

Reply:
Hi @bseeger Will do. I'll add the line, amend the commit, and push to my branch.

Reply:
I tested the inclusion of this edit AND whether it was having the desired effect. Looks good on both accounts!

Reply:
Great! Thanks for testing, @noahwsmith.  Once the comment is added, let's merge.